selfservice/strategy/oidc/strategy_test.go

```diff
@@ -484,6 +484,42 @@ func TestStrategy(t *testing.T) {
 		actual := sr.Methods[identity.CredentialsTypeOIDC]
 		assert.EqualValues(t, expected.Config.FlowMethodConfigurator.(*oidc.FlowMethod).HTMLForm, actual.Config.FlowMethodConfigurator.(*oidc.FlowMethod).HTMLForm)
 	})
+
```

Comment:
This test should be located in `driver/configuration/provider_viper_test.go` :wink:

Reply:
Moved and rewrote that test.